runtime/browser/ui_runtime.cc

```diff
@@ -109,6 +157,30 @@ void UiRuntime::OnAppControl(app_control_h app_control) {
   common::AppDB* appdb = common::AppDB::GetInstance();
   appdb->Set(kAppDBRuntimeSection, kAppDBRuntimeBundle,
              appcontrol->encoded_bundle());
+
+  bool reset_webapp = false;
+  NativeWindow::Type type;
+  if (AlwaysTopRequested(appcontrol.get()) &&
+      HasAlwaysOnTopCategory(app_data_->category_info_list()->categories)) {
+    if (native_window_->type() != NativeWindow::Type::NOTIFICATION) {
+      type = NativeWindow::Type::NOTIFICATION;
+      reset_webapp = true;
+    }
+  } else {
+    if (native_window_->type() != NativeWindow::Type::NORMAL) {
+      type = NativeWindow::Type::NORMAL;
+      reset_webapp = true;
+    }
+  }
+
+  if (reset_webapp) {
+    ResetWebApplication(type);
+    // TODO(t.iwanek): there is still problem with rendering content if
+    // window is replaced ewk won't render page.
+    // This looks the same as for workaround removed in: PR #80
+    native_window_->Show();
```

Comment:
It doesn't work for me on image 20160610.4. If window is created for a second time then content is not rendered and window not shown therefore.